src/kata7.getEmployerRole.js

```diff
@@ -1,5 +1,11 @@
 const getEmployerRole = (employeeName, employees) => {
-
+  let selectedEmployee;
+	employees.forEach(obj => {
```

Comment:
Need to work on indentation in here. This could also be better achieved with [find](https://gitlab.com/mcrcodes/curriculum/blob/master/bytes/javascript-101/arrays/find.md).